intel_extension_for_transformers/neural_chat/examples/finetuning/multi-modal/conversation_utils.py

```diff
@@ -0,0 +1,248 @@
+import dataclasses
+from enum import auto, Enum
+from typing import List, Tuple
+
+
+class SeparatorStyle(Enum):
+    """Different separator style."""
+    SINGLE = auto()
+    TWO = auto()
+    MPT = auto()
+    PLAIN = auto()
+    LLAMA_2 = auto()
+
+
+@dataclasses.dataclass
+class Conversation:
```

Comment:
I think it would be better to reuse [neuralchat's prompt API](https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/neural_chat/prompts/prompt.py) which is based on fastchat's Conversation.

Reply:
good comments.